charts/corda-dev-prereqs/templates/kafka/kafka-statefulset.yaml

```diff
@@ -107,6 +111,15 @@ spec:
                 path: "tls.key"
               - key: "ca.crt"
                 path: "ca.crt"
-        - name: "kafka-data"
-          persistentVolumeClaim:
-            claimName: {{ include "corda-dev-prereqs.kafkaName" . | quote }}
\ No newline at end of file
+  volumeClaimTemplates:
+    - metadata:
+        name: "kafka-data"
+        labels:
+          {{- include "corda-dev-prereqs.labels" . | nindent 10 }}
```

Comment:
Helm isn't really dealing in JSON/YAML - it just deals in text. In particular, functions just return text. So `nindent` is just indenting a block of text. We could switch everything to use JSON rather than YAML so we didn't care about indentation. I've never seen a chart take that approach and you'd have to convert any input YAML. I suspect it would also mean more boilerplate in the templating.